Dockerfile

```diff
@@ -21,8 +26,12 @@ ENV LIBRARY_PATH=/lib:/usr/lib
 # use --build-arg REBUILD=$(date) to invalidate the cache and upgrade all
 # packages
 ARG REBUILD=1
-RUN chmod a+x /start.sh ;\
-    export DEBIAN_FRONTEND=noninteractive \
+RUN set -ex \
+    && groupadd -r matrix \
+    && useradd -r -d /data -M -g matrix matrix \
```

Comment:
I **think** its better to generate the user at start time of container because we can configure the UID of this user and therefore we are more flexible with file ownership. The name doesn't need configurable, only the UID.